src/app/shared/footer/footer.component.html

```diff
@@ -4,25 +4,25 @@
 
     <h3>Join Us</h3>
 
-    <form class="mc-trial row">
+    <!-- <form class="mc-trial row">
```

Comment:
@mashhoodr this quick `join us` used to work. So are we taking it off completely in favor of the sign-up module?